modules/internal/ChapelDomain.chpl

```diff
@@ -1512,12 +1512,16 @@ module ChapelDomain {
       and block domains.
     */
     pragma "no copy return"
-    @unstable("tryBuildArray() is subject to change in the future.")
-    proc tryBuildArray(type eltType, param initElts=true) throws {
+    @unstable("tryCreateArray() is subject to change in the future.")
+    proc tryCreateArray(type eltType) throws {
+      if !this.isDefaultRectangular() && !this.isBlock() then
+        compilerError("'tryBuildArray' is only supported on " +
```

Comment:
This still says 'tryBuildArray', rather than 'tryCreateArray'.

Reply:
Also, might as well change the "doi" method names.

Reply:
I wasn't sure about the `doi` method names, since those are essentially copy-pastes of the `dsiBuildArray` method, so keeping the `build` naming there seemed more in line with that. With that in mind, do you still prefer switching that to `doiCreateArray`?

Reply:
I think the behavior is sufficiently different to warrant the name change to ``doiTryCreateArray``. That way it's the same name at every level and there's less potential confusion when discussing or searching for the relevant methods.